.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: PHP Tests
+
+on: push
+
+jobs:
+  test:
+    name: WP ${{ matrix.wp }} and PHP ${{ matrix.php }}
+    # Ubuntu-20.x includes MySQL 8.0, which causes `caching_sha2_password` issues with PHP < 7.4
+    # https://www.php.net/manual/en/mysqli.requirements.php
+    # TODO: change to ubuntu-latest when we no longer support PHP < 7.4
+    runs-on: ubuntu-16.04
+    continue-on-error: ${{ matrix.allowed_failure }}
+
+    env:
+      WP_VERSION: ${{ matrix.wp }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
+        wp: [ '5.6', '5.7' ]
+        allowed_failure: [ false ]
+        include:
+          - php: '8.0'
```

Comment:
Just wondering why WP 5.6 and WP 5.7 under PHP 8.0 are allowed to fail, given that PHP 8.0 is the current stable version of PHP.

Reply:
@nielslange - yeah, @mikeyarce suggested it here https://github.com/Automattic/Edit-Flow/pull/635#issuecomment-825211538 and I myself think that makes sense until this issue is fixed: https://github.com/Automattic/Edit-Flow/issues/638